src/mps_utils.jl

```diff
@@ -9,6 +9,8 @@ using Missings
 using Random
 using JSON3
 
+using CUDA
+using cuTENSOR
```

Comment:
Why use CUTENSOR directly and not ITensorGPU?

Reply:
I dropped ITensorGPU because most documentation seems to be written for the merged ITensor monorepo, so this was the most accessible way for me to write this code.

I used `using cuTENSOR` to enable the cuTENSOR extension to `CUDA.jl`, but perhaps this is not how Julia works.